<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> QMTest remote target test
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:qmtest%40codesourcery.com?Subject=Re%3A%20QMTest%20remote%20target%20test&In-Reply-To=%3Cba8dd2f90908241707kde26f6ue7f0fa2c8adba115%40mail.gmail.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   
   
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>QMTest remote target test</H1>
    <B>Alok Nagdev</B> 
    <A HREF="mailto:qmtest%40codesourcery.com?Subject=Re%3A%20QMTest%20remote%20target%20test&In-Reply-To=%3Cba8dd2f90908241707kde26f6ue7f0fa2c8adba115%40mail.gmail.com%3E"
       TITLE="QMTest remote target test">alok.nagdev at gmail.com
       </A><BR>
    <I>Tue Aug 25 00:07:07 UTC 2009</I>
    <P><UL>
        
        
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#1351">[ date ]</a>
              <a href="thread.html#1351">[ thread ]</a>
              <a href="subject.html#1351">[ subject ]</a>
              <a href="author.html#1351">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>Hi,

I am a beginner with QMTest and trying to run a test on remote target
but get the following error:

$ qmtest run time_test
Traceback (most recent call last):
  File &quot;../qmtest&quot;, line 132, in ?
    exit_code = main()
  File &quot;../qmtest&quot;, line 110, in main
    exit_code = command.Execute()
  File &quot;C:\Python24\Lib\site-packages\qm\test\cmdline.py&quot;, line 796, in Execute
    return method()
  File &quot;C:\Python24\Lib\site-packages\qm\test\cmdline.py&quot;, line 1700,
in __ExecuteRun
    if engine.Run():
  File &quot;C:\Python24\Lib\site-packages\qm\test\execution_engine.py&quot;,
line 269, in Run
    target.Start(self.__response_queue, self)
  File &quot;C:\Python24\Lib\site-packages\qm\test\classes\process_target.py&quot;,
line 148, in Start
    child_pid = e.Spawn(arg_list)
  File &quot;C:\Python24\Lib\site-packages\qm\executable.py&quot;, line 183, in Spawn
    startupinfo)[0]
TypeError: The object is not a PySTARTUPINFO object

Test file is just one C++ file. This is how I created the target:

$ ../qmtest create-target -a host=192.168.xx.xxx -a
remote_shell=/usr/bin/ssh  -a
database_path=$MY_HOME_DIR/dist_testing/qmtest-2.4.1/scripts/tdb1 a
rsh_target.RSHTarget

Test runs fine on the local machine. I believe it has something to do
with my setup but I am not sure what is causing it. SSH pipe works
fine as I have set up the connection with automatic login. Is there a
test demo file which would give me some clue?  Just for the sake of
experimenting I tried the same with explicitly providing the context
file in the argument list as well but the error didn't change. Thanks
much for your help in advance!

Thanks,
Alok

</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	
	
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#1351">[ date ]</a>
              <a href="thread.html#1351">[ thread ]</a>
              <a href="subject.html#1351">[ subject ]</a>
              <a href="author.html#1351">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="http://sourcerytools.com/cgi-bin/mailman/listinfo/qmtest">More information about the qmtest
mailing list</a><br>
</body></html>
